database/src/main/resources/db/migration/V1_123__add_value_to_external_interface_type_enum.sql

```diff
@@ -0,0 +1,3 @@
+-- Add 'on-behalf-errand' and 'customer-account-info' to interface-data-content
+ALTER TYPE interface_data_content ADD VALUE 'on-behalf-errand';
+ALTER TYPE interface_data_content ADD VALUE 'customer-account-info';
```

Comment:
You can add both values using one insert (or alter)

->

ALTER TYPE interface_data_content 
  ADD VALUE 'on-behalf-errand',
  ADD VALUE 'customer-account-info';